tests/tests/swfs/visual/cache_as_bitmap/text/test.toml

```diff
@@ -1,9 +1,18 @@
 num_frames = 4
 
-[image_comparisons.output]
-tolerance = 0
+[[image_comparisons.output.checks]]
+max_outliers = 12000
```

Comment:
That seems extremly high?

Reply:
It's 1.3%, plus it's one of 2 checks, the other being 0.17%.

The difference comes from the fact that we're rendering device fonts differently. Especially wrt. pixel snapping.

Reply:
Added a comment